src/Microsoft.Health.Fhir.Api/Controllers/ControlPlaneController.cs

```diff
@@ -0,0 +1,84 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Health.ControlPlane.Core.Features.Persistence;
+using Microsoft.Health.ControlPlane.Core.Features.Rbac;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.Health.Fhir.Api.Controllers
+{
+    // TODO: Remove before PR
+    [Route("Admin/IdentityProvider/")]
+    public class ControlPlaneController : Controller
```

Comment:
Used this for testing purposes. This is fully functional and can be used to build on for the api story. Based on the feedback, I can either keep this or remove before merge.

Reply:
Removed this file.